### PR TITLE
Update podgccontroller.go so that users can know the default value of Pod GC threshold

### PR DESCRIPTION
**Pull request by mysunshine92**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
users can know the default vaule for terminated-pod-gc-threshold
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by mysunshine92** [comment]
cc @liggitt

File: cmd/kube-controller-manager/app/options/podgccontroller.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func (o *PodGCControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		return
 	}
 
-	fs.Int32Var(&o.TerminatedPodGCThreshold, "terminated-pod-gc-threshold", o.TerminatedPodGCThreshold, "Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled.")
```
Comment (liggitt) [inline cmd/kube-controller-manager/app/options/podgccontroller.go:36]: thanks for the PR, but the default is already output in help text automatically:

```
kube-controller-manager --help | grep terminated-pod-gc-threshold
```

> --terminated-pod-gc-threshold int32   Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled. (default 12500)

**Pull request closed without merging**